src/frontend/stylesheets/Auth.css

```diff
@@ -0,0 +1,90 @@
+.login__outer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+  line-height: 1.3;
+  width: 80%;
+  height: 80vh;
+}
+
+.login {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: auto;
+  padding: 2rem 3rem;
+  box-shadow: var(--secondary-color) 0px 0px 7px;
+  border-radius: 15px;
+}
+
+.signup__outer {
+  margin-top: 1rem;
+  margin-bottom: 3.5rem;
+}
+
+.error__display {
+  color: red;
+}
+
+.matched__display {
+  color: green;
+}
```

Comment:
try to avoid using hardcode color value and also use color name as hex value

Reply:
as of now, I am using these colors just once that's why I didn't create a separate variables for them
I've changed those values to hex code